### ISSUE TITLE
No scroll bar in manage integrations

### ISSUE THREAD

**Issue by lukebarnard1** [vector-im/riot-web#4846]

**Comment by akontsevich** [comment]
Yeah, good to have scrollbars in "Add integrations" dialog: it was not clear to me Riot has more applications available than 6 visible on first start. Only mouse or keyboard scrolling shows me there are more applications there.
